src/NUnitEngine/nunit.engine/Services/DefaultTestRunnerFactory.cs

```diff
@@ -57,6 +57,7 @@ public override ITestEngineRunner MakeTestRunner(TestPackage package)
             // then defer to the AggregatingTestRunner, which will make
             // the decision on a file by file basis so that each project
             // runs with its own settings.
+            // TODO: What about when ProcessModel is Parallel?
```

Comment:
Do you think we need to handle this case now? Seems like we could address it later down the road when someone needs this functionality.